pb_app/pubspec.yaml

```diff
@@ -7,20 +7,20 @@ environment:
   sdk: ">=2.18.0 <3.0.0"
 
 dependencies:
-  flutter:
-    sdk: flutter
-
-  collection:
+  collection: null
   cupertino_icons: ^1.0.2
   figma_squircle: ^0.5.3
+  flutter:
+    sdk: flutter
+  image_picker: ^0.8.5+3
   pocketbase: ^0.4.0
-  provider:
+  provider: null
```

Comment:
careful using plugins to automatically sort these, leave `flutter` at top and sort them manually. I like using `sort descending` tool in VSCode